### PR TITLE
891562: Need to move the latest changes to public repo

### PR DESCRIPTION
**Pull request by BharatRamsf3693**
Description:
The latest changes to public repo was moved.

### CODE REVIEW

File: server/AppointmentPlanner/Shared/MainLayout.razor [hunk 1]
```diff
@@ -161,6 +161,6 @@
 
     private void showSideBar()
     {
-        open = !open;
+        open = true;
```
Comment (rajendranr-5483) [inline server/AppointmentPlanner/Shared/MainLayout.razor:164]: Breaking reason?

File: server/AppointmentPlanner/wwwroot/lodash.min.js [hunk 2]
```diff
@@ -1,139 +1,140 @@
...
-u&&(q.prototype[t]=function(){var t=this.__wrapped__,o=e?[1]:arguments,f=t instanceof Bt,c=o[0],l=f||yh(t),s=function(n){var t=u.apply(q,a([n],o));return e&&h?t[0]:t};l&&r&&"function"==typeof c&&1!=c.length&&(f=l=!1);var h=this.__chain__,p=!!this.__actions__.length,_=i&&!h,v=f&&!p;if(!i&&l){t=v?t:new Bt(this);var g=n.apply(t,o);return g.__actions__.push({func:Qo,args:[s],thisArg:Y}),new H(g,h)}return _&&v?n.apply(this,o):(g=this.thru(s),_?e?g.value()[0]:g.value():g)})}),r(["pop","push","shift","sort","splice","unshift"],function(n){
-var t=hl[n],r=/^(?:push|sort|unshift)$/.test(n)?"tap":"thru",e=/^(?:pop|shift)$/.test(n);q.prototype[n]=function(){var n=arguments;if(e&&!this.__chain__){var u=this.value();return t.apply(yh(u)?u:[],n)}return this[r](function(r){return t.apply(yh(r)?r:[],n)})}}),ee(Bt.prototype,function(n,t){var r=q[t];if(r){var e=r.name+"";yl.call(is,e)||(is[e]=[]),is[e].push({name:t,func:r})}}),is[Ju(Y,hn).name]=[{name:"wrapper",func:Y}],Bt.prototype.clone=Gt,Bt.prototype.reverse=Ht,Bt.prototype.value=Jt,q.prototype.at=Js,
+/**
```
Comment (rajendranr-5483) [inline server/AppointmentPlanner/wwwroot/lodash.min.js:1]: The changes are not valid

**Review by rajendranr-5483 — APPROVED** [review]

File: server/AppointmentPlanner/Shared/MainLayout.razor (hunk 1 above)
Comment (BharatRamsf3693) [reply server/AppointmentPlanner/Shared/MainLayout.razor:164]: https://github.com/essential-studio/blazor-showcase-appointment-planner/blob/2616911a1aa2e5e3a04e91bfc061ca53d7ebf033/server/AppointmentPlanner/Shared/MainLayout.razor#L164 - I don't know the reason of this change, I have just moved the changes from latest changes having release/26.1.1 repo to public repo.

File: server/AppointmentPlanner/wwwroot/lodash.min.js (hunk 2 above)
Comment (BharatRamsf3693) [reply server/AppointmentPlanner/wwwroot/lodash.min.js:1]: The unnecessary lodash.min.js files were removed: https://github.com/syncfusion/blazor-showcase-appointment-planner/pull/28

**Pull request merged**